libmy/b64_encode.c

```diff
@@ -73,11 +73,6 @@ int base64_encode_block(const char* plaintext_in, int length_in, char* code_out,
 			*codechar++ = base64_encode_value(result);
 			
 			++(state_in->stepcount);
-			if (state_in->stepcount == CHARS_PER_LINE/4)
-			{
-				*codechar++ = ' ';
```

Comment:
Also note that this change breaks the unit tests (which can be run with "make check") as there are still base64 encoded strings in t/test-rdata.c that contain spaces.